lib/model/query/form-attachments.js

```diff
@@ -117,7 +117,7 @@ const createVersion = (xml, form, savedDef, itemsets, publish = false) => ({ Blo
 
 
 ////////////////////////////////////////////////////////////////////////////////
-// CRUD
+// UPDATING
```

Comment:
The file as a whole does CRUD, but as of right now, this particular section only contains the `update()` method.